File: Minimum Consective Array/Minimum.py
def min_consecutive_flips(array):
    flip_count = 0
    current = array[0]
    
    for num in array[1:]:
        if num != current:
            flip_count += 1
            current = num
    
    return flip_count

# Example usage:
array = [1, 0, 0, 1, 0, 1, 1, 0]
result = min_consecutive_flips(array)
print("Minimum consecutive flips needed:", result)

File: Minimum Consective Array/Minimum.java
public class Main {

    public static int minConsecutiveFlips(int[] array) {
        int flipCount = 0;
        int current = array[0];

        for (int i = 1; i < array.length; i++) {
            if (array[i] != current) {
                flipCount++;
                current = array[i];
            }
        }

        return flipCount;
    }

    public static void main(String[] args) {
        int[] array = {1, 0, 0, 1, 0, 1, 1, 0};
        int result = minConsecutiveFlips(array);
        System.out.println("Minimum consecutive flips needed: " + result);
    }
}
